lua/gitlab/actions/summary.lua

```diff
@@ -83,6 +83,7 @@ M.summary = function()
   end)
 
   git.current_branch_up_to_date_on_remote(vim.log.levels.WARN)
+  git.check_mr_in_good_condition()
```

Comment:
It's nice to have this warning here as well, in case the user is just looking at the summary outside of the reviewer pane.